draft-ferrieuxhamchaoui-quic-lossbits.md

```diff
@@ -87,17 +89,46 @@ With QUIC, the equivalent transport headers are encrypted and passive packet
 loss observation is not possible, as described in {{TRANSPORT-ENCRYPT}}.
 
 Measuring TCP loss between similar endpoints cannot be relied upon to
-evaluate QUIC loss.  QUIC could be routed by the network differently and
+evaluate QUIC loss. QUIC could be routed by the network differently and
 the fraction of Internet traffic delivered using QUIC is increasing every
 year.  It is imperative to measure packet loss experienced by QUIC users
 directly.
 
-Since explicit path signals are preferred by {{!RFC8558}}, two explicit loss
+## On-path loss observation
+
+There are three sources of observable loss that network operators need to 
```

Comment:
Maybe "loss observation" rather than "observable loss", since we, as operators, don't really _need_ (observable) losses :D

Reply:
loss observation indeed